vendor.mod

```diff
@@ -0,0 +1,95 @@
+module github.com/docker/cli
+
+// We use a specially crafted vendor.mod to exclusively vendor with go modules
+// so that all dependencies that do use go.mod can be easily consumed. The mod
+// file is not named go.mod because it implies opting in for all the rules
+// around SemVer, which this repo cannot abide by as it uses CalVer.
```

Comment:
@AkihiroSuda @errordeveloper Let me know if it sounds good to you.

Reply:
Looks good, thanks

Reply:
Thank you! Looks good, but perhaps it could even be shorter, like instead of  "We use..." you could say just "This file is for ..., so that ...".